server/controllers/admin.py

```diff
@@ -70,6 +70,26 @@ def index():
     return render_template('staff/index.html', courses=courses)
 
 
+@admin.route("/client/<name>", methods=['GET', 'POST'])
+@is_staff()
+def client_version(name):
+    version = Version.query.filter_by(name=name).one()
+
+    form = forms.VersionForm(obj=version)
+    if form.validate_on_submit():
+        model = version
```

Comment:
Good point. I think I copied the structure from `new_assignment`. I'll make the change now.